docs/banzai/index.rst

```diff
@@ -151,59 +187,116 @@ has dimensions that are half of the full image).
-the data will not be reduced and an exception will be raised.
 
+Header Sanity
+=============
+The header sanity test first checks if any principal FITS header keywords are either missing or set to ``'N/A'``.
+The following keywords are checked:
+``RA``, ``DEC``, ``CAT-RA``, ``CAT-DEC``,
+``OFST-RA``, ``OFST-DEC``, ``TPT-RA``,
+``TPT-DEC``, ``PM-RA``, ``PM-DEC``,
+``CRVAL1``, ``CRVAL2``, ``CRPIX1``,
+``CRPIX2``, and ``EXPTIME``.
 
-Source Detection
-================
-Source detection uses the "Source Extraction in Python" (SEP; https://github.com/kbarbary/sep).
-This is similar to Source Extractor, but is written purely in Python and Cython. This allows more
-customization.
+This routine then verifies that the RA value (``CRVAL1``) is between 0 and 360
+and that the declination value (``CRVAL2``) is between -90 and 90.
 
-We estimate the background by taking a 3x3 median filter of the image and the doing a 32x32 block
-average of the image.
+Finally, the header checker ensures that exposure time value (``EXPTIME``) is greater than 0.
+Note that this final test is not performed on bias frames, which can sometimes have negative
+exposure time values.
 
-We use the default match filter for source detection that is provided by SEP.
 
-We do aperture photometry using an elliptical aperture that is set by 2.5 times the Kron radius. This
-produces approximately the same results as FLUX_AUTO from SExtractor.
+Thousands Test
+==============
+There is a known issue with the Sinistro cameras where a large fraction of pixels report values of exactly 1000.
+This test measures the fraction of 1000-valued pixels in each Sininstro frame, and if this fraction is above
+20%, the frame is rejected.
 
-We set the source detection limit at 3 times the global rms of the image. MINAREA is set to 5,
- the default. This should minimize false detections, but may miss the faintest sources.
 
-The catalog is returned as the 'CAT' as fits binary table extension of the final science image. The catalog
-has the following columns: the position in pixel coordinates, (X, Y), the flux (Flux), the error in the flux
-(Fluxerr), the semi-major and semi-minor axes (a, b), and the position angle (theta).
+Saturation Test
+===============
+A pixel is considered saturated if its values is greater than the ``SATURATE`` header kewyword.
+This test measures the fraction of saturated pixels in each Sininstro frame, and if this fraction is above
+5%, the frame is rejected.
 
 
-Astrometry
-==========
-The WCS is found by using Astrometry.net (Lang et al. 2009, arXiv:0910.2233). We use the catalog from
-the source detection (the previous step) as input.
+Pattern Noise Detector
+======================
+Occasionally some cameras have been found to exhibit highly structured electrical pattern noise. Although it
+is not a common occurrence, it is still desirable to detect the issue as soon as possible.
```

Comment:
Mention that this usually is indicative of a failing camera?

Reply:
Done